<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>PostgreSQL server details : </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblUsername">
       <property name="minimumSize">
        <size>
         <width>58</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>58</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Username : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="usernameEdit">
       <property name="text">
        <string>postgres</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPassword">
       <property name="minimumSize">
        <size>
         <width>53</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>53</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Password :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordEdit">
       <property name="text">
        <string>dbpass</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lblHostname">
       <property name="minimumSize">
        <size>
         <width>58</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>58</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Host :  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hostEdit">
       <property name="text">
        <string>localhost</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPort">
       <property name="minimumSize">
        <size>
         <width>53</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>53</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Port : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portEdit">
       <property name="text">
        <string>5432</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="dbcheckButton">
     <property name="text">
      <string>Check connection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>91</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>91</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>PostgreSQL path : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="psqlButton">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="psqlEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Be sure to select the folder for the exact version you're running so instead of &quot;C:Program Files\PostgreSQL&quot; </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>you must select &quot;C:Program Files\PostgreSQL9.3&quot;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
